com.unity.render-pipelines.core/Runtime/Utilities/Blit.hlsl

```diff
@@ -0,0 +1,209 @@
+#ifndef UNITY_CORE_BLIT_INCLUDED
+#define UNITY_CORE_BLIT_INCLUDED
+
+#include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl"
+#include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Color.hlsl"
+#include "Packages/com.unity.render-pipelines.core/ShaderLibrary/UnityInstancing.hlsl"
+
+TEXTURE2D_X(_BlitTexture);
+TEXTURECUBE(_BlitCubeTexture);
+SamplerState sampler_PointClamp;
+SamplerState sampler_LinearClamp;
+SamplerState sampler_PointRepeat;
+SamplerState sampler_LinearRepeat;
+uniform real4 _BlitScaleBias;
+uniform real4 _BlitScaleBiasRt;
+uniform real _BlitMipLevel;
+uniform real2 _BlitTextureSize;
+uniform uint _BlitPaddingSize;
+uniform int _BlitTexArraySlice;
+
+#if SHADER_API_GLES
+struct Attributes
+{
+    real4 positionCS       : POSITION;
+    real2 uv               : TEXCOORD0;
+    UNITY_VERTEX_INPUT_INSTANCE_ID
+};
+#else
+struct Attributes
+{
+    uint vertexID : SV_VertexID;
+    UNITY_VERTEX_INPUT_INSTANCE_ID
+};
+#endif
+
+struct Varyings
+{
+    real4 positionCS : SV_POSITION;
+    real2 texcoord   : TEXCOORD0;
```

Comment:
You don't have enough precision to store uv in half precision.

Reply:
Yes, you are correct. I think that's actually true for the position as well as we use it as dst rect. I reverted the whole float -> real change. It was a bit of rash last minute change from me anyway.